ManatiCR/artavia
# Feature/ar 7 home section frontend

**Steps to test**

- Run `sudo rm -r web && composer update && ./scripts/artavia_local_install.sh`
- Go to homepage

See the main slider
